src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -494,155 +494,155 @@
-    <sys:String x:Key="Hash_Checker_XXH128.Description">Dis/abilita il calcolo XXH128.</sys:String>
-    <sys:String x:Key="Hash_Checker_MD5.Description">Dis/abilita il calcolo MD5.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA1.Description">Dis/abilita il calcolo SHA-1.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA256.Description">Dis/abilita il calcolo SHA-256.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA384.Description">Dis/abilita il calcolo SHA-384.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA512.Description">Dis/abilita il calcolo SHA-512.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA3_256.Description">Enable or disable SHA3-256 computing, only supported on Windows 11 build 25324 or later.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA3_384.Description">Enable or disable SHA3-384 computing, only supported on Windows 11 build 25324 or later.</sys:String>
-    <sys:String x:Key="Hash_Checker_SHA3_512.Description">Enable or disable SHA3-512 computing, only supported on Windows 11 build 25324 or later.</sys:String>
-    <sys:String x:Key="Hash_Checker_Rows_Per_Page">Rows per page</sys:String>
-    <sys:String x:Key="Hash_Checker_Rows_Per_Page.Description">Set the number of files displayed per page in the file list.</sys:String>
+    <sys:String x:Key="Hash_Checker_AutoCompute.Description">Ad importazione del file completata avvia automaticamente il calcolo del valore hash .</sys:String>
+    <sys:String x:Key="Hash_Checker_CRC32.Description">Abilita/disabilita calcolo CRC-32.</sys:String>
+    <sys:String x:Key="Hash_Checker_XXH32.Description">Abilita/disabilita calcolo XXH32.</sys:String>
+    <sys:String x:Key="Hash_Checker_XXH64.Description">Abilita/disabilita calcolo XXH64.</sys:String>
+    <sys:String x:Key="Hash_Checker_XXH3.Description">Abilita/disabilita calcolo XXH3.</sys:String>
+    <sys:String x:Key="Hash_Checker_XXH128.Description">Abilita/disabilita calcolo XXH128.</sys:String>
+    <sys:String x:Key="Hash_Checker_MD5.Description">Abilita/disabilita calcolo MD5.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA1.Description">Abilita/disabilita calcolo SHA-1.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA256.Description">Abilita/disabilita calcolo SHA-256.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA384.Description">Abilita/disabilita calcolo SHA-384.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA512.Description">Abilita/disabilita calcolo SHA-512.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_256.Description">Abilita/disabilita elaborazione SHA3-256, supportata solo in Windows build 25324 o successiva.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_384.Description">Abilita/disabilita elaborazione SHA3-384, supportata solo in Windows build 25324 o successiva</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_512.Description">Abilita/disabilita elaborazione SHA3-512, supportata solo in Windows build 25324 o successiva</sys:String>
```

Comment:
Line 561 and 562 missing trailing period.

Reply:
Fixed (Add "." at the end of lines 561/562)